src/TotkRstbGenerator/Commands/RstbCommands.cs

```diff
@@ -12,9 +12,28 @@ public class RstbCommands
         """;
 
     [Command("generate", Aliases = ["gen", "calculate", "calc"], Description = "Calculate the RSTB values for a target RomFS folder")]
-    public async Task Generate([Argument] string romfs, [Option("output", ['o'], Description = GEN_OUTPUT_DESC)] string? output, [Option("padding", ['p'], Description = "Adds padding to every generated RSTB value")] uint padding = 0)
+    public async Task Generate([Argument] string romfs, string[]? rstbs, [Option("output", ['o'], Description = GEN_OUTPUT_DESC)] string? output, [Option("padding", ['p'], Description = "Adds padding to every generated RSTB value")] uint padding = 0)
     {
-        RstbGenerator generator = new(romfs, output, padding);
+        RstbGenerator generator = new(romfs, null, output, padding);
         await generator.GenerateAsync();
     }
+
+    [Command("patch", Description = "Calculate the RSTB values for a input rstbs")]
+    public async Task Patch([Argument] string romfs, [Argument] string[] rstbs, [Argument] string output, [Option("padding", ['p'], Description = "Adds padding to every generated RSTB value")] uint padding = 0)
```

Comment:
`output` should be optional and use `romfs` as the default like the `gen` command.

Reply:
Ok